src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -237,13 +237,14 @@ public List<String> validateJurisdictionCodes(CaseData caseData) {
         List<String> errors = new ArrayList<>();
         List<VettingJurCodesTypeItem> codeList = caseData.getVettingJurisdictionCodeCollection();
         if (!codeList.isEmpty()) {
-            codeList.stream().filter(codesTypeItem -> caseData.getJurCodesCollection().stream()
-                    .map(existingCode -> existingCode.getValue().getJuridictionCodesList())
-                    .collect(Collectors.toList()).stream()
-                    .anyMatch(code -> code.equals(codesTypeItem.getValue().getEt1VettingJurCodeList())))
-                .forEach(c -> errors
-                    .add(String.format(ERROR_EXISTING_JUR_CODE, c.getValue().getEt1VettingJurCodeList())));
-
+            if (caseData.getJurCodesCollection() != null && !caseData.getJurCodesCollection().isEmpty()) {
```

Comment:
Minor thing: You can use org.apache.commons.collections4.CollectionUtils.isNotEmpty(caseData.getJurCodesCollection()). This will check both null and isEmpty.